joomla/joomla-cms
# Fix for Issue https://github.com/joomla/joomla-cms/issues/41270 Loading Views

Fix like suggested in https://github.com/joomla/joomla-cms/issues/41270#issuecomment-1655905244

Pull Request for Issue #41270 .

### Testing Instructions
make a copy of
html/com_content/category/default.php
html/com_content/category/default.xml
html/com_content/category/default_articles.php
html/com_content/category/default_children.php

rename them and put in this template an alternate layout for a category list:
html/com_content/category/mylayout.php
html/com_content/category/mylayout.xml
html/com_content/category/mylayout_myarticles.php
html/com_content/category/mylayout_mychildren.php

In mylayout.php change articles to myarticles (I did that but did not mentio in the test instructions)
Change something in the _myarticles file or put a die(); at the top to see if your template is loaded.
Change the layout title in the mylayout.xml so you recognize it in the menu item types overview

Create a new menu item and select my layout as a menu item type


### Actual result BEFORE applying this Pull Request
Is not working when you load multiple sublayouts and your childtemplate name is the same name as the main template

### Expected result AFTER applying this Pull Request
Is working when you load multiple sublayouts and your childtemplate name is the same name as the main template


### Link to documentations
Please select:
- [x ] No documentation changes for docs.joomla.org needed

- [x ] No documentation changes for manual.joomla.org needed
